ush/forecast_postdet.sh

```diff
@@ -868,7 +864,7 @@ GSD_in() {
 
 GSD_nml() {
 	echo "SUB ${FUNCNAME[0]}: Creating name list for GSD"
-	sh parsing_namelists_GSD.sh
+	# generate GSD-specific input
```

Comment:
should this whole section just be deleted now?

Reply:
Agreed. I was not sure if we wanted to leave it as a template for future development.

Reply:
Commit 6b19a39b6f086ff7eb077c60a3acbc40d6dae5a1 removes such GSD-specific shell functions.